rns_shell/platform/graphics/x11/WindowX11.cpp

```diff
@@ -21,7 +21,9 @@
 #include "WindowContextFactory.h"
 
 #include "x11/PlatformDisplayX11.h"
-
+#define X11KeyRelease 0
+#define X11KeyPress   1
```

Comment:
I think we need to define RNS_KEY_Press and Release enum for this and use in both X11 & WPE backend